### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by bshyong**

### CODE REVIEW

File: src/applications/personalization/profile360/vet360/components/AddressField/index.jsx [hunk 1]
```diff
@@ -16,6 +16,80 @@ import Vet360ProfileField from '../../containers/ProfileField';
...
+
+export const validateCleanData = ({ addressLine1, city, stateCode,  internationalPostalCode, zipCode, countryName }, property) => {
+  const isInternational = inferAddressType(countryName, stateCode) === ADDRESS_TYPES.INTERNATIONAL;
+  const validateAll = !property;
+
+  return {
+    addressLine1: (property === 'addressLine1' || validateAll) && !addressLine1 ? 'Street address is required' : '',
+    city: (property === 'city' || validateAll) && !city ? 'City is required' : '',
+    stateCode: (property === 'stateCode' || validateAll) && !isInternational && !stateCode ? 'State is required' : '',
+    zipCode: (property === 'zipCode' || validateAll) && !isInternational && !zipCode ? 'Zip code is required' : '',
```
Comment (cehsu) [inline src/applications/personalization/profile360/vet360/components/AddressField/index.jsx]: This should be 'ZIP code' instead of 'Zip code'.

Comment (cehsu) [inline src/applications/personalization/profile360/vet360/components/AddressField/index.jsx]: Should the ZIP format (5-9 digits) be validated?

File: src/applications/personalization/profile360/vet360/components/AddressField/index.jsx [hunk 2]
```diff
@@ -16,6 +16,80 @@ import Vet360ProfileField from '../../containers/ProfileField';
...
+};
+
+export const validateCleanData = ({ addressLine1, city, stateCode,  internationalPostalCode, zipCode, countryName }, property) => {
+  const isInternational = inferAddressType(countryName, stateCode) === ADDRESS_TYPES.INTERNATIONAL;
+  const validateAll = !property;
+
+  return {
+    addressLine1: (property === 'addressLine1' || validateAll) && !addressLine1 ? 'Street address is required' : '',
+    city: (property === 'city' || validateAll) && !city ? 'City is required' : '',
+    stateCode: (property === 'stateCode' || validateAll) && !isInternational && !stateCode ? 'State is required' : '',
```
Comment (cehsu) [inline src/applications/personalization/profile360/vet360/components/AddressField/index.jsx]: Is state displayed as a select? If not, should the state code value be validated, i.e. checked from a list of valid options?

**Review by cehsu — APPROVED** [review]
LGTM. I had some minor questions/comments for your consideration.

File: src/applications/personalization/profile360/vet360/components/AddressField/index.jsx (hunk 2 above)
Comment (bshyong) [reply src/applications/personalization/profile360/vet360/components/AddressField/index.jsx]: thanks! yes these are loaded from a constants file for a list of valid options in a dropdown select

Comment (cehsu) [reply src/applications/personalization/profile360/vet360/components/AddressField/index.jsx]: Thanks for explaining, it seems like the forms library could be helpful here eventually.

File: src/applications/personalization/profile360/vet360/components/AddressField/index.jsx (hunk 1 above)
Comment (bshyong) [reply src/applications/personalization/profile360/vet360/components/AddressField/index.jsx]: Yep we will add in more validations in the next iteration—there is also an address validator that sits in front of submissions and will reject addresses that don't exist

**Pull request merged**